src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+function checkComments(comments) {
```

Comment:
no need to make this check. just map comments in your jsx

Reply:
done